ocaml/xapi/cert_distrib.ml

```diff
@@ -245,3 +266,37 @@ let go ~__context ~from_hosts ~to_hosts ~existing_cert_strategy =
   List.iter
     (fun host -> Worker.remote_regen_bundle host rpc session_id)
     to_hosts
+
```

Comment:
The names `from_host` and `to_host` are a bit odd, `source` and `destination` (`src`, `dst`) feel more natural to me.

Reply:
I think it makes the code to be read in a quite natural way so I tried to keep that style.